packages/machine/src/ethereum/eth-virtual-app-agreement-commitment.ts

```diff
@@ -54,7 +53,11 @@ export class ETHVirtualAppAgreementCommitment extends MultiSendCommitment {
       data: iface.functions.delegateTarget.encode([
         {
           registry: this.networkContext.AppRegistry,
-          terms: this.terms!,
+          terms: {
+              assetType: AssetType.ETH,
+              limit: new BigNumber(0),
```

Comment:
The limit is 0? Really?

Also, using `Zero` would remove needing to `new BigNumber`